src/sentry/static/sentry/app/views/organizationGroupDetails/groupTags.jsx

```diff
@@ -35,14 +37,20 @@ class GroupTags extends React.Component {
     this.fetchData();
   }
 
+  componentDidUpdate(prevProps) {
+    if (!isEqual(prevProps.environments, this.props.environments)) {
+      this.fetchData();
+    }
+  }
+
   fetchData = () => {
-    const {api, group} = this.props;
+    const {api, group, environments} = this.props;
+    const env_query = environments.map(e => '&environment=' + e).join('');
     this.setState({
       loading: true,
       error: false,
     });
-
-    api.request('/issues/' + group.id + '/tags/', {
+    api.request('/issues/' + group.id + '/tags/?' + env_query, {
```

Comment:
This is old code, but can we use template strings here? 

```suggestion
    api.request(`/issues/${group.id}/tags/`, {
```